### ISSUE TITLE
Unable to download station data from Met Data Portal

### ISSUE THREAD

**Issue by NatashaCowie** [pacificclimate/pdp#318]
I'm trying to help a Ministry of Transportation employee retrieve precipitation data observations for a specific one-week time period. I directed them to the Met Data Portal and attempted to take them through the steps to download data, but neither one of us was able to retrieve any data. The downloads do not seem to initiate at all, there is no response. One of my colleagues tried this as well. We've attempted downloads on Chrome, Firefox, and Edge.

We ARE able to download station metadata, but cannot download timeseries or climatology.

My IP address is 142.36.47.5
Attempts have been made over the last hour, about 2:30-3:30pm PDT

I have been trying a few different combinations of filters:
- Start date: 2024-08-15
- End date: 2024-08-21
- Include stations with no observations: CHECKED (have also tried UNCHECKED)
- Only include stations with climatology : UNCHECKED (have also tried CHECKED)
- Network: Have tried all networks and a single network at a time
- Variable: Have tried with all selected, with one selected, and with a few selected
- Observation frequency: Have tried with only one selected (Hourly) or all selected
- Map selection: Have tried drawing a polygon around only one, or just a few, stations to narrow the selection

Station Data tab:
- Clip time series to filter date range: CHECKED (have also tried UNCHECKED)
- Have tried all three output formats (NetCDF, CSV/ASCII, and MS Excel 2010)
- Have tried to download timeseries and climatology

Thank you!

**Comment by Nospamas** [comment]
Hi @NatashaCowie

I've restarted our downloader backend and the portal which seems to have brought it back to life. While testing however I did notice that we seem to have attained some data we can't currently handle via the downloader. This data is isolated to the `RT` network. 

As such I recommend:
- Deselect the `RT` network
- Try your download as normal again

I'll work with the team to get that other data back in working order as soon as I can.

Thanks for the report.
